content/docs/user-guide/components/reference/atom/ssao.md

```diff
@@ -5,16 +5,25 @@ description: 'Open 3D Engine (O3DE) SSAO component reference.'
 toc: true
 ---
 
-The **SSAO** component approximates indirect lighting in a scene by using the *screen space ambient occlusion* technique.
+The **SSAO** (screen-space ambient occlusion) component approximates the occlusion of light in a scene using the depth buffer. This occlusion is sometimes referred to as contact-shadows. 
 
+{{< image-width "/images/user-guide/components/reference/atom/ssao-bistro.png" "700" "Image of SSAO applied to the Bistro scene" >}}
 
-## Provider ##
+## Properties
 
-[Atom Gem](/docs/user-guide/gems/reference/rendering/atom/atom/)
+{{< image-width "/images/user-guide/components/reference/atom/ssao-component.jpg" "500" "SSAO Component Properties" >}}
 
+| Property | Description |
+|-|-|
+| **Enable SSAO** | Whether SSAO should be enabled or not (note that SSAO is enabled by default - even if there is no SSAO component in the level - and an SSAO component must be added and this value set to false in order to disable SSAO). |
+| **SSAO Strength** | The strength of the SSAO effect - the higher this value the darker the SSAO will appear. |
+| **Sampling Radius** | The sampling radius of the SSAO effect in screen UV space - decrease/increase this value to have a more local/global SSAO effect. |
+| **Enable Blur** | Whether a blur is applied to the computed SSAO buffer. Enabling with result in a smoother looking SSAO image. |
+| **Blur Strength** | How strong of a blur to apply to the SSAO. Weaker values will result in a sharper albeit more noisy image. |
+| **Blur Sharpness** | The SSAO blur tries to respect geometric edges to avoid smearing the SSAO across edges or objects. Increasing this value will result in a sharper looking blur, whereas decreasing it will result in a softer looking SSAO blur. |
+| **Blur Edge Threshold** | An edge threshold can be specified for the blur to ignore depth changes of a certain value. This has the most effect at small values. |
```

Comment:
"can be specified" - Does that mean we can not choose to not activate this, even with blur enabled? 

Current suggestion to rephrase assumes that it's activated if blur is enabled. And I assume that at `0.0`, it has the most effect?
```suggestion
| **Blur Edge Threshold** | An edge threshold, in which the blur ignores depth changes at the specified value. This has the most effect with smaller values. |
```